.github/actions/setup-e2e-tools/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'setup-e2e-tools'
+description: 'Set up required tooling for creating an e2e test environment'
+
+runs:
+  using: "composite"
+  steps:
+    - name: 'Install Skaffold'
```

Comment:
There is a [GitHub action](https://github.com/marketplace/actions/skaffold-github-action) on the marketplace for installing and configuring Skaffold which might make this more flexible in the future. It might be overkill for this use case but I would suggest you take a look at it.

Reply:
I was going to make the same comment. I'm not sure why we're downloading this ourselves, and I'm _very_ worried about always downloading "latest".

Reply:
Thanks Brad and Seth for your recommendations here. Frankly, I was also slightly skeptical in start but after I did pilot for WISE i realize it is going to be a common pattern where client may need simple wrapper to set up all their required tooling. 

It is two steps now but which may not add much of the value and probably a documentation with some copy / paste it can be solved. But then we anticipate it to grow to 10+ and maintaining cost of version / steps will go up if different client has to do it themselves.

The github marketplace recommendation seems great and we can definitely switch to it.   The underlying minikube action is a good example of defining such reusability which install and start the minikube with some defaults.